basis/io/sockets/secure/unix/unix.factor

```diff
@@ -14,10 +14,10 @@ M: ssl-handle handle-fd file>> handle-fd ;
 
 M: unix socket-handle fd>> ;
 
-M: secure ((client)) ( addrspec -- handle )
-    addrspec>> ((client)) <ssl-socket> ;
+M: secure ((client)) ( secure -- handle )
+    [ hostname>> ] [ addrspec>> ] bi ((client)) <ssl-socket> ;
```

Comment:
It's weird that some words are ``( hostname addrspec -- )`` and others are ``( addrspec hostname -- )``...?